packages/chord-mark/tests/unit/renderer/components/renderLine.spec.js

```diff
@@ -80,3 +80,81 @@ describe.each([
 		});
 	});
 });
+
+describe.each([
+	[
+		'line only',
+		{
+			shouldOpenSection: false,
+			shouldClosePriorSection: false,
+			shouldCloseFinalSection: false,
+			sectionWrapperClasses: [],
+		},
+		'^<p class="cmLine">%l</p>$',
```

Comment:
I'm not sure why we want a regexp and a placeholder here, I would go with the full expected result, e.g. `<p class="cmLine">myLine</p>`

Reply:
Yep. I promise there was a reason, and then it got factored out and I didn't just change it back to a string.